controllers/articles.controller.js

```diff
@@ -8,4 +11,9 @@ function getArticleById(request, response, next) {
     .catch(next);
 }
 
-module.exports = { getArticleById };
+function getAllArticles(request, response, next) {
+  return selectAllArticles()
+    .then((rows) => response.status(200).send({ articles: rows }))
```

Comment:
you can use something more descriptive here than rows.

Reply:
Refactored as part of PR https://github.com/dku7/be-nc-news/pull/16